ReactQt/runtime/src/linkingmanager.cpp

```diff
@@ -34,6 +34,13 @@ void LinkingManager::canOpenURL(const QString& url,
     resolve(d->bridge, QVariantList{true});
 }
 
+void LinkingManager::getInitialURL(const ModuleInterface::ListArgumentBlock& resolve,
+                                   const ModuleInterface::ListArgumentBlock& reject) {
+    Q_D(LinkingManager);
+    Q_UNUSED(reject);
+    resolve(d->bridge, QVariantList{QString()});
```

Comment:
Should return null instead of an empty string based on [the documentation](https://facebook.github.io/react-native/docs/linking#getinitialurl).

Reply:
@madhavarshney, good point, thanks!